subsys/bluetooth/mesh/rpl.c

```diff
@@ -45,6 +45,8 @@ static inline int rpl_idx(const struct bt_mesh_rpl *rpl)
 	return rpl - &replay_list[0];
 }
 
+static void clear_rpl(struct bt_mesh_rpl *rpl);
```

Comment:
Is it possible to reorder the static functions so that this forward declaration is not necessary? It's usually only in circular dependency situations where we have to use forward declarations.